users/serializers.py

```diff
@@ -29,6 +34,11 @@
     """,
     flags=re.I | re.VERBOSE | re.MULTILINE,
 )
+NAME_CONTAINS_HTML_URL_MSG = (
+    "Full name can not contain HTML or URL. Please try a different one."
+)
```

Comment:
If we show the validation character message in this case as well, the would be fine. We don't need a new message for it.